zbus/tests/basic.rs

```diff
@@ -112,6 +112,15 @@ fn fdpass_systemd() {
     f.metadata().unwrap();
 }
 
+#[cfg(target_os = "linux")]
+#[cfg(feature = "ibus")]
+#[test]
+fn test_ibus() {
+    let ibus = block_on(conn::Builder::ibus().unwrap().build()).unwrap();
```

Comment:
this reminds me, you need to also add the blocking variant of the method in `blocking::connection::Builder`. Then use that here.